### ISSUE TITLE
How to search documents that start with a specific search term?

### ISSUE THREAD

**Issue by Aaron23145** [typesense/typesense#187]
I am trying to find documents that certain field starts with the received search term. I was wondering if this is possible in Typesense.

For example: Having the `alu` search term, I'm expecting to receive the documents that have `aluminium` in an specific field.

Checking the documentation I only found examples to filter by exact matches besides the default behavior. Is there a way to do this in Typesense currently?

**Comment by jasonbosco** [comment]
@Aaron23145 This is controlled by the `prefix` search parameter: https://typesense.org/docs/0.18.0/api/#search-collection. It defaults to `true` and should find prefix matches out of the box.

If it's not working as described, could you post your schema and your search query?

**Comment by Aaron23145** [comment]
@jasonbosco  This issue happened to me when I was on a device with Ubuntu 19.10, with the version 0.18.0 of Typesense.

Now I have installed Typesense in another device with Ubuntu 20.04 LTS, the same version of Typesense and the same dataset, and everything worked as expected.

Could be that I made a mistake or that it was actually a bug from Typesense. Not sure for now, since I can't access that device right now. When I'm able to verify this I'll reply again to confirm it.
